test/fixtures.rb

```diff
@@ -226,3 +226,38 @@ def routing_helper_path
     routes.path(:dashboard)
   end
 end
+
+class LinkTo
+  include Lotus::Helpers::LinkToHelper
+
+  class Routes
+    def self.path(name, id = nil)
+      if id
+        "/#{ name }/#{id}"
+      else
+        "/#{ name }"
+      end
+    end
+  end
+
+  def routes
+    Routes
+  end
+
+  def link_to_posts
+    link_to('Posts', routes.path(:posts))
+  end
+
+  def link_to_post
+    link_to('Post', routes.path(:post, 1))
+  end
+
+  def link_to_with_class
+    link_to('Post', routes.path(:posts), class: 'first')
+  end
+
+  def link_to_with_id
+    link_to('Post', routes.path(:posts), id: 'posts_link')
```

Comment:
What do you think about changing `posts_link` to BEM syle `posts__link`? Pleaser read more at http://www.sitepoint.com/bem-smacss-advice-from-developers/ - It's for the sake of encouraging good frontend practices